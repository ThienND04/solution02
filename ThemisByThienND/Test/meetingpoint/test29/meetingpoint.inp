414 764 35
212 305 484024
353 192 168590
380 255 753668
385 351 12542
398 167 205220
113 52 885028
340 209 601540
99 404 760350
343 211 782767
136 188 192829
342 261 671750
260 216 465418
330 81 200558
353 321 188308
326 336 70324
239 13 959103
42 134 32196
97 67 22027
266 79 88043
20 169 65782
159 97 806052
94 361 185555
88 130 100780
200 33 787601
402 284 200221
281 234 655634
413 100 235416
166 338 85908
120 388 58873
263 59 77463
409 392 202277
323 329 5168
138 97 12142
281 189 764275
340 195 10877
206 93 129383
121 158 783051
322 166 91249
349 324 544828
151 1 31892
378 192 895282
107 172 41133
392 207 285190
5 283 537161
294 96 358943
391 324 622667
367 1 223836
315 347 163451
203 162 520917
349 256 489511
308 380 206730
47 278 640010
334 15 804125
65 235 216923
366 350 732948
36 343 790794
3 413 3048
96 133 391469
152 117 80757
261 260 127854
31 315 966410
223 76 862846
285 66 236486
146 230 647079
44 104 104659
379 33 949986
347 187 29434
190 51 703168
291 120 741354
207 221 9721
183 104 705516
91 247 596947
365 319 728395
282 335 706480
26 100 701572
341 45 4331
193 196 117025
371 236 423852
297 181 844688
184 150 589956
224 53 878217
14 355 163069
410 371 632862
323 112 667822
131 310 538035
207 153 8038
48 172 24919
270 152 118112
202 205 551312
241 395 466146
207 42 8423
226 149 105572
82 38 191887
333 403 243906
296 263 711434
128 219 213651
244 131 573820
189 181 95406
411 24 713791
232 258 187083
209 101 630438
256 373 489866
321 124 52980
109 41 693864
352 197 215702
349 339 522533
402 368 203149
12 211 20886
7 327 518473
298 248 157267
169 317 17190
196 301 137881
158 179 143729
152 16 31530
144 48 781533
46 116 61617
189 344 638929
161 394 677988
77 302 591733
82 313 8758
153 20 100033
400 18 60074
301 32 80480
59 350 157969
374 354 211157
18 287 607009
355 253 747006
8 184 531925
246 103 668439
229 134 657252
400 168 257019
318 388 695025
286 133 55347
368 266 453550
377 40 154728
378 270 736855
50 132 265430
283 109 495023
14 124 70044
274 340 640369
142 148 72660
328 13 955485
70 71 791304
288 184 813559
375 196 48993
219 9 204357
350 49 75620
202 46 197194
357 313 224318
64 162 757798
357 2 177811
224 48 920698
373 15 222229
50 31 37663
249 335 57825
44 85 80902
89 338 30526
370 93 23790
279 249 681723
103 101 85485
264 354 244558
134 242 139328
249 18 33811
17 411 57771
128 384 21917
363 132 504776
1 39 1335
369 56 69392
285 304 495563
75 160 33536
319 317 36612
48 23 24810
10 125 683185
354 151 38534
66 220 52084
277 53 88899
44 412 202845
296 57 219404
15 313 760730
396 109 82312
150 369 773697
272 150 229381
196 287 453559
354 96 364070
302 181 681617
406 81 30555
257 303 25188
218 25 679296
244 151 172425
223 197 4978
65 241 471703
326 210 908881
218 110 140211
390 299 664722
342 54 86920
31 313 740153
349 233 498622
360 159 141455
196 240 72881
299 96 264098
193 194 54563
342 250 617388
164 120 492641
300 31 165013
174 25 907180
170 191 5945
211 112 89912
380 169 64204
172 303 786709
244 56 129287
370 135 66452
48 211 20081
293 318 8221
255 245 271001
135 325 649153
353 218 27125
332 386 482007
83 143 159496
185 309 749886
87 92 75285
129 185 160480
340 204 111756
11 168 419162
383 101 599946
245 195 563849
308 50 24821
1 26 36492
307 354 76182
141 95 43883
263 337 696797
171 87 795675
175 97 52607
367 18 115356
207 293 3165
218 107 231556
204 241 3540
228 331 815239
214 347 527430
158 26 26362
277 34 767809
193 413 828576
200 66 775229
382 23 244593
379 50 991275
57 1 766832
255 16 828728
362 243 682408
184 274 615739
105 181 695995
409 275 44322
214 348 27178
32 335 40897
389 286 654106
47 152 12578
109 245 443152
406 143 571176
193 161 114602
235 49 853683
362 321 12777
348 310 49413
149 367 127622
44 400 14949
395 37 75556
267 6 509519
400 1 48406
330 369 4565
286 249 628326
200 3 144136
293 15 4354
198 108 753682
249 219 130493
114 97 792939
276 5 19772
329 225 30437
203 144 48612
122 332 729269
231 409 681000
89 12 41009
304 317 601174
196 31 220764
172 409 84809
298 255 27896
394 137 838069
351 13 769570
351 339 21563
341 76 36554
336 1 1194
321 345 731732
76 382 204161
271 32 957326
68 245 52357
31 363 732544
293 30 86920
348 120 488841
53 217 891927
141 55 57948
103 27 24338
215 346 619334
36 131 729988
224 78 196429
172 53 67400
349 234 67729
232 135 757169
148 180 3196
405 120 185522
176 184 123566
398 24 3543
271 36 952744
127 242 29874
237 375 685868
118 409 59396
1 368 294840
93 229 561135
352 215 5259
405 282 172933
73 8 63245
227 7 20635
219 314 538304
406 251 1385
153 13 12670
341 22 243115
344 300 635013
36 173 708938
349 66 186011
230 398 27017
46 126 32124
94 68 7689
28 335 34353
231 320 467811
142 290 486283
207 32 30496
280 126 16354
130 219 60397
234 259 212846
268 43 130580
326 360 883919
117 376 522357
269 212 701375
260 271 735641
142 174 234143
1 369 5638
243 15 743029
276 126 129817
410 165 20461
400 32 4837
302 153 783440
25 316 188916
402 391 73644
345 128 741031
1 27 26027
46 363 683446
255 313 207402
140 93 285785
119 400 34417
9 1 68427
47 33 71093
343 332 580806
223 359 18849
252 129 634990
147 158 14711
247 270 116685
60 261 3552
296 11 261649
102 219 24516
389 43 3481
297 66 894381
67 280 108783
254 250 153181
196 304 527283
345 226 598381
47 389 626
13 289 68272
169 376 560728
245 381 528317
139 212 31143
299 353 86344
93 269 837245
120 288 224580
141 112 23276
369 357 1000000
332 45 197687
52 330 44622
103 25 912795
96 176 121551
368 86 552593
59 43 77345
159 265 510602
140 132 166662
138 128 39062
11 133 101378
313 286 10701
93 314 661281
129 97 85619
67 361 917987
293 178 285032
357 60 871792
296 217 15734
178 167 56877
307 90 122944
201 352 450377
402 74 8636
30 61 103763
358 53 865058
263 69 729106
398 401 215416
51 305 191897
2 88 716221
228 245 238910
24 253 959944
6 156 199493
117 124 193766
317 396 4216
339 219 492923
238 411 237808
375 60 177075
44 308 5132
13 36 30446
408 174 190219
273 112 122848
43 106 870661
137 201 203657
100 390 97056
33 379 949986
80 72 32291
53 291 41573
158 327 754586
238 199 153243
332 236 471822
104 190 120316
108 138 26430
75 5 63926
356 369 200485
304 52 734387
144 325 23996
203 195 615144
330 369 4566
176 323 500966
166 341 169725
108 153 41470
378 349 740257
263 396 82546
119 41 885608
102 407 15899
71 208 138140
101 77 60889
94 97 754459
222 194 83999
88 19 881262
368 233 446956
348 269 697539
326 149 24695
197 82 179083
43 329 699086
293 383 719560
208 266 99093
339 389 649196
44 182 197513
117 258 553336
124 97 7523
46 58 19177
194 367 24368
45 42 38955
137 128 40289
279 217 146488
217 114 169633
178 187 513419
48 18 72385
123 256 426049
163 7 162260
152 128 69327
156 396 798754
99 301 114714
32 320 265606
176 129 219397
21 224 161482
361 29 3402
35 9 5105
85 315 893217
354 329 748654
407 36 265051
26 187 778198
88 184 77771
173 226 555302
284 104 153798
42 412 214805
164 270 39140
408 180 31931
29 99 712584
279 305 54312
31 14 44332
7 409 202982
147 312 189755
352 337 218140
179 331 65194
322 18 34777
132 108 222428
248 359 141775
160 276 10616
295 37 526142
136 84 630723
263 13 257099
315 170 249517
283 207 756521
384 392 217209
268 63 12351
295 146 417664
84 124 98559
227 369 272694
234 26 274412
43 86 727440
306 32 85107
136 404 254376
312 330 257117
397 331 77781
111 364 558618
52 386 685319
20 316 669179
150 296 206901
385 198 371
96 109 159927
103 294 743074
6 188 146049
301 36 75898
251 184 53325
160 359 682514
384 38 900912
173 306 628413
94 69 241520
251 203 526311
323 285 520261
84 242 79006
291 70 157316
101 223 828332
272 60 874866
66 335 27302
50 336 2308
10 15 242224
26 350 2877
72 154 83352
215 21 47110
171 11 232716
393 274 7507
280 93 11068
404 184 809734
113 274 134736
102 226 95515
119 344 758368
4 108 931663
91 291 689835
115 102 532703
311 66 20833
164 326 214198
318 239 952349
23 240 128143
362 279 742803
73 401 529962
146 47 597384
366 9 703889
377 222 501981
180 393 583344
364 115 46336
135 213 542176
47 70 78433
36 207 35078
291 297 914541
321 160 194385
97 1 7930
413 91 246642
263 243 488818
291 15 16411
77 277 29265
330 399 261274
36 314 762940
24 183 808001
247 32 86322
351 377 1823
398 77 157568
78 13 742661
109 237 722221
45 166 174057
344 139 586656
206 303 518692
170 375 447134
97 334 816786
232 127 726745
395 50 742689
45 193 84454
20 194 94403
202 150 491878
299 152 41934
306 162 119423
205 308 810445
238 134 933537
329 184 629701
132 194 69464
297 360 3893
85 139 140177
403 99 771328
368 266 453550
349 176 69770
51 386 221454
177 262 500285
33 398 5619
5 132 36501
85 177 129525
215 119 665376
188 99 698803
324 6 3515
2 13 810125
49 371 191118
372 13 798955
180 52 159440
228 132 687697
413 360 18042
267 138 261924
122 303 189522
67 48 6137
24 174 20351
299 309 833518
94 46 672126
414 35 938789
22 306 179222
373 349 354
144 197 151533
196 375 48993
159 142 523857
260 138 245179
259 14 12558
310 189 37839
38 357 23672
94 333 3797
84 199 719975
331 189 60873
370 293 109781
179 43 86591
401 282 536358
267 312 14675
40 205 101408
95 63 24974
241 11 444542
296 255 2485
185 57 512802
58 87 52189
207 123 293567
295 108 249102
321 56 6595
253 211 939397
127 409 126909
199 375 523063
414 35 938789
65 345 48418
118 168 150044
297 350 912175
171 257 134230
354 237 926364
172 1 11176
78 128 701229
341 61 142139
30 206 176035
47 125 830118
216 11 6083
139 412 18234
218 163 147159
243 36 715471
286 316 3626
332 148 40675
333 226 564400
230 195 87681
61 265 96459
407 238 674028
192 13 70448
341 181 42074
91 19 266031
336 153 3838
58 225 679202
78 84 646351
412 52 181475
117 44 175764
179 347 637539
158 380 172199
38 311 903969
201 161 43575
125 336 944807
373 299 72422
280 205 700028
67 24 5672
65 82 38329
66 380 177890
353 341 191959
334 340 659967
216 270 484095
14 387 879654
100 214 421369
341 157 155674
197 157 748705
25 178 661891
155 110 13891
27 127 196867
244 409 108334
234 248 492654
381 149 93190
191 140 328408
218 123 22771
5 60 98583
173 270 510512
402 151 59797
241 109 5476
130 120 565969
199 363 60277
209 38 215677
143 356 606369
43 389 3481
62 377 34485
175 145 881402
186 172 186060
159 259 715924
138 46 70189
163 74 36381
305 91 42864
51 239 15206
245 140 282126
374 267 414
67 72 101794
13 254 20877
309 259 905859
216 299 560273
54 184 532851
201 310 62979
74 110 43328
411 49 634431
45 111 164600
31 323 772745
384 375 229870
26 163 100215
66 221 34370
88 36 63457
367 98 5347
259 60 35790
305 12 692640
403 147 932525
392 79 538171
410 296 47266
389 353 140230
350 260 225883
187 8 93385
36 271 952744
139 400 206130
183 152 715167
78 37 61384
225 254 750062
292 258 15096
33 121 801113
103 346 78500
312 237 729468
125 57 179169
14 118 69882
32 103 6797
277 93 17666
43 347 724130
396 38 789710
227 353 21590
261 332 120737
74 397 118845
311 331 63392
151806 992967
714237 464149
896410 180981
985426 378937
290939 477916
555469 552031
346042 80866
600849 162703
891918 678564
126685 675877
473959 421746
662442 949688
214658 224182
155026 721953
200217 766091
862077 797985
305453 755751
35770 928957
276959 986757
242097 3051
490786 240349
639809 372527
216425 936808
97517 469572
7456 44937
883778 116461
382721 808513
773327 737409
369137 448609
239746 616309
563458 628670
788804 64505
918059 657447
770796 993557
761688 423733
