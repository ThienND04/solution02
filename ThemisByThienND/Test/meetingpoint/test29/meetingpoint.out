153676401726
435737975561
169902972009
355741887293
294523659419
518240630459
75916111274
152743786667
637028418996
128245885885
395930505594
670603947882
210459595598
156936075346
202683873657
749139540165
309216486413
36210722170
280371411839
3088591371
225636997361
349724249803
219091572425
98718506957
7547865376
109332305729
387436505441
692271457701
373685136977
242699910466
570400366018
60556584445
617204011683
780292977516
397795879337
